jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1098,6 +1098,30 @@ def func1(*jax_args):
 
 tf_impl[lax.scan_p] = _scan
 
+def _sort(*operand: tf.Tensor, dimension: int = -1,
+          is_stable: bool = True) -> Tuple[TfVal, ...]:
+  if len(operand) == 0:
+    raise TypeError("Sort requires at least one operand")
+  if len(operand) > 1:
+    raise NotImplementedError("TODO: multiple tensors")
+  dimension = lax._canonicalize_axis(dimension, len(operand[0].shape))
+  if dimension == lax._canonicalize_axis(-1, len(operand[0].shape)):
+    # TODO: rewrite XlaSort to allow stable sorting.
+    return tfxla.sort(operand)
+  else:
+    # tfxla.sort only supports sorting the last axis of its operands.
+    # TODO: fix XlaSort to allow an axis parameter. In the meantime, we transpose the
+    # tensors before and after sorting if it does not happen on the last axis.
+    shape_range = list(range(len(operand[0].shape)))
+    permutation     = shape_range[ dimension + 1:] + shape_range[: dimension + 1]
+    inv_permutation = shape_range[-dimension - 1:] + shape_range[:-dimension - 1]
```

Comment:
It is somewhat dangerous to add transpositions because they may be expensive. Eventually, we will see if we can fix XlaSort to specify the axis. Meanwhile, maybe we can just raise UnimplementedError. Or, if we do add transposition, there are multiple ways of doing it, e.g, just swap the `dimension` with the last axis.